dev/source/docs/the-ardupilot-autotest-framework.rst

```diff
@@ -7,23 +7,23 @@ Autotest Framework
 .. image:: ../images/autotest.jpg
 
 ArduPilot's AutoTest suite allows for the creation of repeatable tests
-which help prevent regressions in ArduPilot's behaviour.  It is based
+which help prevent regressions in ArduPilot's behavior.  It is based
```

Comment:
I am not english native, but I think both are valid

Reply:
yes American English spells without the "u", UK English includes it...either is fine....will merge as the others are correct ...